Weather Alert/CItyTableViewController.swift

```diff
@@ -0,0 +1,158 @@
+//
+//  CItyTableViewController.swift
+//  Weather Alert
+//
+//  Created by Jack Hopkins on 22/11/2016.
+//  Copyright © 2016 UKFast. All rights reserved.
+//
+
+import UIKit
+import CoreData
+
+class CityTableViewController: UITableViewController {
+    
+    // MARK: Variables
+        
+    var dataController: DataController { return AppShared.instances.dataController }
+    var cities: [City] {
```

Comment:
You do realise that every time you access `cities` you essentially create a new fetch, right?
Not ideal.
If it's a static list, just capture it in a stored variable. If it's dynamic and might change while the app is running, I would suggest using a `NSFetchedResultsController` for constant updates.